README.md

```diff
@@ -78,6 +78,31 @@ This script performs an analysis of two small physics-only UKESM development job
 using the debug suite.
 
 
+Keeping the code up to date
+---------------------------
+
+To keep the code up to date with the main branch, fetch the changes from github:
+```
+git fetch
+```
+
+You can merge the main branch into your local code:
+```
+git merge origin/main
+```
+
+If the `environment.yml` file has changed in the merge, you can update your conda environment with:
+```
+mamba env update -n bgcval2 -f environment.yml
+pip install -e .[develop]
```

Comment:
I have more detailed instructions (and **not** updating the env since that's tricksy to sort out if it craps up) in https://github.com/valeriupredoi/bgcval2/pull/77 - which you should have been notifed for a review for, unless you still have all your GH comms turned off :man_facepalming:

Reply:
So, cut this part?

Reply:
yeah, cut it here, but do have a look at #77 and see if some bits from here can go there (not the updating of envs though, I'd avoid doing that, especially with `mamba update env`, and `conda update env` is slow like a slow train)